storybook/pages/DAppsWorkflowPage.qml

```diff
@@ -25,12 +25,19 @@ import SortFilterProxyModel 0.2
 import AppLayouts.Wallet.panels 1.0
 import AppLayouts.Profile.stores 1.0
 
-import utils 1.0
+import mainui 1.0
 import shared.stores 1.0
+import utils 1.0
 
 Item {
     id: root
 
+    Popups {
```

Comment:
Could you please clarify why is this required, not easy to tell given that there is no reference to it.

Reply:
This is needed to make that "Instructions" popup to open from storybook. Another way would be to explicitly call "open" but then code for app and storybook would further diverge.
I will add a comment to clarify